src/components/CategoryInput.tsx

```diff
@@ -23,23 +19,33 @@ const CategoryInput = ({ setSelectedCategoryInfo }: CategoryInputProps) => {
     fetchAndStoreAllCategories();
   }, []);
 
+  const handleChangeCategory = (categoryName: string) => {
+    if (categoryName === "none") {
+      setSelectedCategoryInfo({
+        id: -1,
+        name: "",
+      });
```

Comment:
this is actually an anti pattern. I assume you are going to check against the id somewhere, but ideally, you want to avoid this and simply set it to `null` if no data is returned or expected. You never know at some point you might receive an item with an ID of `-1`

Reply:
I didn't think about this. I've made the change now and will keep this in mind for future